frame/contracts/src/benchmarking/mod.rs

```diff
@@ -2700,7 +2700,7 @@ benchmarks! {
 	// depends on how much memory is already allocated.
 	#[pov_mode = Ignored]
 	instr_memory_grow {
-		let r in 0 .. 1;
+		let r in 0 .. INSTR_BENCHMARK_BATCHES;
```

Comment:
This won't work as we can't grow the memory that often.

Reply:
Okay, going to revert and test with the other ones for now. Derp.